src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/ConversionsBase.cs

```diff
@@ -733,7 +728,7 @@ public static bool IsBaseClass(TypeSymbol derivedType, TypeSymbol baseType, ref
 
             for (TypeSymbol b = derivedType.BaseTypeWithDefinitionUseSiteDiagnostics(ref useSiteDiagnostics); (object)b != null; b = b.BaseTypeWithDefinitionUseSiteDiagnostics(ref useSiteDiagnostics))
             {
-                if (HasIdentityConversion(b, baseType))
+                if (HasIdentityConversionInternal(b, baseType, IncludeNullability))
```

Comment:
>IncludeNullability [](start = 63, length = 18)

Is it safe to drop this parameter?

Reply:
If we dropped `IncludeNullability`, we'd treat `Base<object?>` as a base class of `Derived<object>`. I've left this as is and changed `IsBaseInterface` above to use `IncludeNullability` for consistency.

---
In reply to: [168036473](https://github.com/dotnet/roslyn/pull/23981#discussion_r168036473) [](ancestors = 168036473)